### ISSUE TITLE
Proposal: Adjust parameters of bandwidth model

### ISSUE THREAD

**Issue by halibobo1205** [tronprotocol/tips#560]
# Simple Summary

Modify the parameters related to the bandwidth model to decrease the volume of low-value transactions in the network.

# Motivation
In recent months, the transaction volume has grown rapidly in the TRON network. The daily network transaction volume exceeds 11 million, about 11 times that of Ethereum. 

Based on statistics and analysis, it is found that there were a large number of low-value or even fraudulent transactions in the network. In addition, some accounts use daily free bandwidth to send large amount of low-value transactions, encroaching on network resources and seriously affecting the development of the entire ecology.

In order to improve the situation, it is proposed to decrease the volume of low-value transactions through the following measure.

- Modify `freenetlimit`  from 1500 to 600, which means the daily free bandwidth for an account changes from 1500 to 600

Appropriately decrease the daily free bandwidth will strike the low-value transactions initiated by large amount of accounts using free bandwidth,  meanwhile, I also hope that other developers in the community could make more ideas and contributions to further mitigate the influence on other transactions in the ecosystem.

# How to initiate the voting request
Modify the daily free bandwidth to 600,
- createProposal 61 600

# Background
At present, the average daily transaction volume on the chain has increased sharply, from about 8 million per day to over 11 million per day in the past month, as shown in the figure below, for details, refer to: https://tronscan.org/#/data/charts/txn/daily-txn

![](https://hackmd.io/_uploads/HkxFVkJd3.png)

Based on statistics, about 75.34% of TRX transfer transactions are low-value transactions in the last 3 months, as shown in the table below. 

|| Daily last week | Daily last month | Daily last 3 months|
| -------- | -------- | -------- |--|
| Total TRX transfer transactions    | 5,054,115    | 4,586,870     | 4,409,105 |
| Transfer transactions with TRX < 0.1     | 4,006,043     | 3,559,799     | 3,321,912 |
| Proportion     | 79.26%    | 77.61%     | 75.34% |

And around 44.86% of smart contract triggers are low-value transactions in the last 3 months, and it still delivers an increasing trend, as shown in the table below.

|| Daily last week | Daily last month | Daily last 3 months|
| -------- | -------- | -------- |--|
| Volume of smart contract triggers | 5,244,816    | 3,478,842    | 2,618,377 |
| Volume of low-value transactions     | 3,661,780 | 2,026,656 | 1,174,615 |
| Proportion | 69.82% | 58.26% | 44.86% |

# Analysis
A comparison showing the relationship between free bandwidth and free transactions is displayed in the table below, 

| Daily free bandwidth | Daily number of free TRX transfers | Daily number of free TRC10 transfers | Daily number of free TRC20 transfers | Daily number of free TRC20 transferfroms |
| :---: | :---: | :---: | :---: | :---: |
| 1500 | 5.60 | 5.26 | 3.97 | 4.35 |
| 700 | 2.61 | 2.46 | 1.85 | 2.03 |
| 600 | 2.24 | 2.11 | 1.59 | 1.74 |
| 500 | 1.87 | 1.75 | 1.32 | 1.45 |

After the daily free bandwidth is set to 600, it will still support a user sending 2 free TRX/TRC10 transactions, or sending 1 free TRC20 transaction.

**Comment by tronenergymarket** [comment]
There are several parts that doesn't fit:
- comparison vs ethereum is a big error, many devs went to L2 because of their fees, Why do we want to be like them?
- the focus is wrong imho, you should not focus on mitigate transactions but to make transactions have a less impact on nodes (which I imagine is the motivation of the proposal)
- if you change what made Tron different and attractive to other blockchain it will lose it's attractive (low cost/free tx)

--------------------------------------------

If there is an issue with the maintenance of the nodes the focus must be there. 
- How can we improve the nodes to support more tx?
- Is maintaining all the historic of tx sustainable?
- Should we cut the tail of the chain to allow nodes be more sustainable?

**Comment by Ademillery** [comment]
> There are several parts that doesn't fit:
> 
> * comparison vs ethereum is a big error, many devs went to L2 because of their fees, Why do we want to be like them?
> * the focus is wrong imho, you should not focus on mitigate transactions but to make transactions have a less impact on nodes (which I imagine is the motivation of the proposal)
> * if you change what made Tron different and attractive to other blockchain it will lose it's attractive (low cost/free tx)
> 
> If there is an issue with the maintenance of the nodes the focus must be there.
> 
> * How can we improve the nodes to support more tx?
> * Is maintaining all the historic of tx sustainable?
> * Should we cut the tail of the chain to allow nodes be more sustainable?

Wow, first of all we should make consensus of the main purpose of this draft. In the motivation part, what I see from the description is that the main issue is too many phishing or fraudulent transactions in network, which is consistent with my daily experience, rather than too much work load for nodes. With the volume of transactions for now, the network is still smooth, fast and steady as before, so I think this is only an incidental problem.

In fact, I also wish the cost of transaction could be lower and I even tried to submit a proposal, but as for this draft, I would honestly say it is not intend to tax more, on the contrary, it would be effective way to kill trash transactions. Because the cost of bandwidth only takes very small part of a transaction compared to energy. By increasing the cost of bandwidth, it will have little impact on normal transactions truly as described in article, but the trash transactions they use free bandwidth and staked resource to send millions transactions. Increasing cost of bandwidth may have little impact on each of them too, but we should know these transactions are made by few users, to regard these transactions as a whole, it will drastically increase their cost to make such amount of transactions. 

Please I wish TRON could take actions to trash transactions earlier, to prevent tragic story of users losing assets from happening.

**Comment by Jackmrshen** [comment]
Agreed, and another fact is that I can see the effort of dapps like tronscan and tronlink that they have hidden majority of fraudulent transactions, but it can not fundamentally solve the problem with so much transactions in nodes. Even the capability of nodes is scaled up and refined, the fraudulent transactions could be more easily scaled up to occupy the most resource soon again. 

The trash transactions flood should be punished, this approach is once and for all.

**Comment by MannGates** [comment]
What are low-value transactions? I have not seen other blockchain limiting the value of assets.

**Comment by tronenergymarket** [comment]
> 

From time ago I don't know why Tron must act as police in the blockchain. That's mainly against any decentralized purpose and it's just hurting real users on the blockchain. We have seen it happening with energy which the main purpose was to defeat fraudulent / low-value transactions and we got into a monster that make most of the dapps devs dissapear.

Again I'm against any purpose of continuously being police and not focusing on improving the blockchain. Fight vs scammers is a lost battle which should be fought with other tools like education imho. It should not be the main purpose of Tron to fight low-value tx. It's like saying that the purpose of Tron is to not be used.

This way of thinking (with all due respect) lacks of big picture mindset.

Also this is not going to improve anything in the blockchain and will make it less attractive to any dapp dev. (burning more TRX doesn't improve or make the blockchain more attractive since last moves with USDD)

I don't know why the goal in the last 2-3 years has been to make all transactions more expensive when it should be the other way.

**Comment by brooklynn1212** [comment]
> What are low-value transactions? I have not seen other blockchain limiting the value of assets.

@MannGates the so called low-value transactions mentioned here refers to the <0.1 TRX transfer transactions or fake TRC20 transactions, in fact small transaction is not limited on TRON, and you can still transfer small value assets, this proposal will not increase obvious cost. Just that if someone tries to batch send large amount of abuse transactions, it will cost them much more.

**Comment by LauraYH** [comment]
Not all so-called low-value transactions are scamming/phishing, but that's pretty much the gist of them. With the crypto markets' cooling down, it seems fraudulence activities take the part. TRON transaction volume maintains steady growth, do the math. 

I think the community definitely needs to take some countermeasures against this trend, most victims are newcomers, probably valuable users with assets to invest during the bad market period. I say just terminate the free daily bandwidth, sharing the same mechanism as the energy consumption. The free perk has urged lots of dummy addresses controlled by some dudes who are doing malicious things and making good profits.

**Comment by Ocea91** [comment]
> What are low-value transactions? I have not seen other blockchain limiting the value of assets.

<img width="340" alt="image" src="https://github.com/tronprotocol/tips/assets/114645811/44f11dc6-84a4-401c-bddc-777af5c6dd60">
Here is a glimpse of the low values, and you will have the picture of this TIP.

**Comment by souppopnix** [comment]
I think the main purpose here is to reduce those phishing transactions or low value transactions, let's just say it's ironic that they were booming recently, I believe many had been scammed by those counterfeit USDT tokens, it's very annoying to see them in my transaction history. 

To some extent, limit free bandwith can help with this issue, though doing it too much will definitely affect us, ordinary TRON users. `totalnetlimit` on the other hand, can slow down those scammers while doing no harm as we use the free quota and won't occupy large amount of bandwiths. Raising the unit price will increase their cost but it's the same for us, it depends on how much is `appropriate`. **Appropriately** increasing the bandwidth costs will not have a relatively large impact on normal transactions. It should be admitted that increasing bandwidth fees can effectively curb a large number of low value or fraudulent transactions, thereby reducing network resource occupancy and serious impacts on ecological development.

I urge/hope the community to further discuss this proposal, and think about the subsequent parameter adjustments to achieve the goal of reducing scam transactions without affecting normal transactions.

**Comment by sunflower-sun** [comment]
From the perspective of the development of the whole ecology, there SHOULD NOT be so many low value/ fraudulent transactions. Why so many low value transactions occur, the one reason is the transactions on TRON can be completely free, and fraudsters can send transactions at zero cost. The other is the popularity of TRON, they can use the banner of Tron  to attract users easily, but if the users are come into TRON at this way, they will misunderstand TRON and be disappointed to  TRON, this is not conducive to the development of TRON. 

So expected more reasonable parameters of bandwidth model.

**Comment by Ademillery** [comment]
> > Wow, first of all we should make consensus of the main purpose of this draft. In the motivation part, what I see from the description is that the main issue is too many phishing or fraudulent transactions in network, which is consistent with my daily experience, rather than too much work load for nodes. With the volume of transactions for now, the network is still smooth, fast and steady as before, so I think this is only an incidental problem.
> > In fact, I also wish the cost of transaction could be lower and I even tried to submit a proposal, but as for this draft, I would honestly say it is not intend to tax more, on the contrary, it would be effective way to kill trash transactions. Because the cost of bandwidth only takes very small part of a transaction compared to energy. By increasing the cost of bandwidth, it will have little impact on normal transactions truly as described in article, but the trash transactions they use free bandwidth and staked resource to send millions transactions. Increasing cost of bandwidth may have little impact on each of them too, but we should know these transactions are made by few users, to regard these transactions as a whole, it will drastically increase their cost to make such amount of transactions.
> > Please I wish TRON could take actions to trash transactions earlier, to prevent tragic story of users losing assets from happening.
> 
> From time ago I don't know why Tron must act as police in the blockchain. That's mainly against any decentralized purpose and it's just hurting real users on the blockchain. We have seen it happening with energy which the main purpose was to defeat fraudulent / low-value transactions and we got into a monster that make most of the dapps devs dissapear.
> 
> Again I'm against any purpose of continuously being police and not focusing on improving the blockchain. Fight vs scammers is a lost battle which should be fought with other tools like education imho. It should not be the main purpose of Tron to fight low-value tx. It's like saying that the purpose of Tron is to not be used.
> 
> This way of thinking (with all due respect) lacks of big picture mindset.
> 
> Also this is not going to improve anything in the blockchain and will make it less attractive to any dapp dev. (burning more TRX doesn't improve or make the blockchain more attractive since last moves with USDD)
> 
> I don't know why the goal in the last 2-3 years has been to make all transactions more expensive when it should be the other way.
> 
> Please stop this nonsense.

Hey dude, emotionally I support you, having similar feelings with you that TRON should keep its advantage of low transactions cost, and I always advocated lower fees if you check my comments in history.

But data tells truth. If you check the data on tronscan this time, you will find out that with active accounts almost unchanged, the daily transactions rise terribly. If no one cares the transaction but only to improve the performance of chain, soon this chain will become a trash chain overwhelming normal transactions and dapps. So this time I am with this proposal to limit bandwidth supplying to scammers with too cheap cost. And of course I will look closely to the parameters, I will be in first place to change to an opponent once I find it harms normal users.

And indeed the big picture is that we should welcome devs with good dapps rather than scammers, this is sustainable.

**Comment by tronpulse** [comment]
@halibobo1205 can you provide data on the numbers of low value transactions made with the 1500 free bandwidth ?

**Comment by halibobo1205** [comment]
> @halibobo1205 can you provide data on the numbers of low value transactions made with the 1500 free bandwidth ?

@tronpulse  According to the data in recent 1 month, 4,013,854 low-value transactions from over 1 million accounts use free bandwidth per day.

**Comment by tronpulse** [comment]
> > @halibobo1205 can you provide data on the numbers of low value transactions made with the 1500 free bandwidth ?
> 
> @tronpulse According to the data in recent 1 month, 4,013,854 low-value transactions from over 1 million accounts use free bandwidth per day.

So almost 100% of low transactions are from those accounts ? Setting free bandwidth to 0 is enough to tackle the issue.

**Comment by halibobo1205** [comment]
> > > @halibobo1205 can you provide data on the numbers of low value transactions made with the 1500 free bandwidth ?
> > 
> > 
> > @tronpulse According to the data in recent 1 month, 4,013,854 low-value transactions from over 1 million accounts use free bandwidth per day.
> 
> So almost 100% of low transactions are from those accounts ? Setting free bandwidth to 0 is enough to tackle the issue.

@tronpulse Not 100% of the low-value transactions use free bandwidth only, most low-value TRX transfer transactions use free bandwidth. In addition, in last month, 2,026,656 low-value TRC20 transactions and in last week 3,661,780 low-value TRC20 transactions happened, majority of which used staked bandwidth, and part of them used bandwidth from TRX burning.

**Comment by txoh1603** [comment]
I believe the free bandwidth is designated to encourage and attract newcomers and users to participate, and it has become something to take advantage of, especially for those controlling a large number of addresses, and utilizing free bwd to promote scams and phishing URLs. 

The daily bwd should be shaved off some bit to stop or at least mitigate the circumstance. This setup should not be a part of malicious behaviors on chain.

**Comment by Blonde488** [comment]
> > What are low-value transactions? I have not seen other blockchain limiting the value of assets.
> 
> <img alt="image" width="340" src="https://user-images.githubusercontent.com/114645811/247462418-44f11dc6-84a4-401c-bddc-777af5c6dd60.png">
> Here is a glimpse of the low values, and you will have the picture of this TIP.

**Comment by ChuffinCelery72** [comment]
> I believe the free bandwidth is designated to encourage and attract newcomers and users to participate, and it has become something to take advantage of, especially for those controlling a large number of addresses, and utilizing free bwd to promote scams and phishing URLs.
> 
> The daily bwd should be shaved off some bit to stop or at least mitigate the circumstance. This setup should not be a part of malicious behaviors on chain.

@txoh1603  I partially agree with your opinion, it is true that free bandwidth has been taken advantage of, scammers use a lot of addresses to spam people, sending _virus_ links. 

But it is also worth considering that the free bandwidth has been a very unique feature of TRON, allowing new users to initiate transactions and trigger contracts even without any assets, think about when we were new on this chain. Therefore, free bandwidth should not be completely canceled, that would be a lost for all of us. Reducing the free amount is acceptable, this feature needs to be properly retained.

**Comment by txoh1603** [comment]
> > I believe the free bandwidth is designated to encourage and attract newcomers and users to participate, and it has become something to take advantage of, especially for those controlling a large number of addresses, and utilizing free bwd to promote scams and phishing URLs.
> > The daily bwd should be shaved off some bit to stop or at least mitigate the circumstance. This setup should not be a part of malicious behaviors on chain.
> 
> @txoh1603 I partially agree with your opinion, it is true that free bandwidth has been taken advantage of, scammers use a lot of addresses to spam people, sending _virus_ links.
> 
> But it is also worth considering that the free bandwidth has been a very unique feature of TRON, allowing new users to initiate transactions and trigger contracts even without any assets, think about when we were new on this chain. Therefore, free bandwidth should not be completely canceled, that would be a lost for all of us. Reducing the free amount is acceptable, this feature needs to be properly retained.

This feature is for newcomers, and now is kind of in the way of joining the network even though the original mindset of this perk is very thoughtful. I agree to save some fuel for new users, but 1500 is too much. TRX transferring usually cost around 270 unit of bandwidth, maybe retaining two to three transferring cost maximum as free daily bandwidth.
@ChuffinCelery72

**Comment by tronpulse** [comment]
> > I believe the free bandwidth is designated to encourage and attract newcomers and users to participate, and it has become something to take advantage of, especially for those controlling a large number of addresses, and utilizing free bwd to promote scams and phishing URLs.
> > The daily bwd should be shaved off some bit to stop or at least mitigate the circumstance. This setup should not be a part of malicious behaviors on chain.
> 
> @txoh1603 I partially agree with your opinion, it is true that free bandwidth has been taken advantage of, scammers use a lot of addresses to spam people, sending _virus_ links.
> 
> But it is also worth considering that the free bandwidth has been a very unique feature of TRON, allowing new users to initiate transactions and trigger contracts even without any assets, think about when we were new on this chain. Therefore, free bandwidth should not be completely canceled, that would be a lost for all of us. Reducing the free amount is acceptable, this feature needs to be properly retained.

If not completly removed. You can just create more accounts to get more free bandwidth.

**Comment by txoh1603** [comment]
@tronpulse You are right, that is fatality for them, but it will increase the complexity and cost of their running. Like @ChuffinCelery72 said, this feature needs to be retained. So the countermeasure would be a combination of free bandwidth, unit cost and the total supply of it.

**Comment by brooklynn1212** [comment]
I made a comparison showing the relationship between free bandwidth and free transactions, please check it in the table below, 

| Daily free bandwidth | Daily number of free TRX transfers | Daily number of free TRC10 transfers | Daily number of free TRC20 transfers |
| :---: | :---: | :---: | :---: |
| 1500 | 5.60 | 5.26 | 3.97 |
| 700 | 2.61 | 2.46 | 1.85 |
| 600 | 2.24 | 2.11 | 1.59 |
| 500 | 1.87 | 1.75 | 1.32 |

I think it would be enough for users including newbies to create 1 free transaction, which means cut the free bandwidth to 500 daily. This will decrease the low-value transactions to the most, while keep it possible for new comers to test a transaction on TRON.

**Comment by txoh1603** [comment]
@brooklynn1212  500 per day would be enough to keep this feature for the new. And one free TRX transfer per day would make it several times more difficult for the scams to spread in the network.

**Comment by MannGates** [comment]
Oh if that's the case, cut the daily free bandwidth is reasonable and acceptable. The most annoying transactions are small TRX and TRC10 transactions. The TRC20 transactions are possible to be controlled by education and warnings, I don't agree excessive preference for new users at the cost of losing the interests of old users.

TRON is famous for its low cost, increasing bandwidth price or shrink the bandwidth supply will harm all users. @tronpulse  is correct, free bandwidth can be reduced, we do not need other measures. As for creating more accounts, as far as I know, now activating a new account also costs TRX, that would be a way to stop creating giant accounts for free bandwidth.

**Comment by amiramir88** [comment]
According to proposal #483 , Energy unit price of energy was raised to 420 sun to  inhibit those low-value transactions. Few weeks later Dynamic Energy was implemented and Justlend rental Energy created  and this result in energy rent price very low on the different energy marketplaces. My question did this proposal helped inhibiting low value trc20  transactions ? Since scammers can buy now cheaper energy.?

Before  proposal #483, energy was being sold around 120 sun / day.  Now its being sold for 60 sun / day or even less.

Now concerning this proposal #560 , is Justlend planning to create a Bandwith Rental platform in the future?

**Comment by MannGates** [comment]
> According to proposal #483 , Energy unit price of energy was raised to 420 sun to inhibit those low-value transactions. Few weeks later Dynamic Energy was implemented and Justlend rental Energy created and this result in energy rent price very low on the different energy marketplaces. My question did this proposal helped inhibiting low value trc20 transactions ? Since scammers can buy now cheaper energy.?
> 
> Before proposal #483, energy was being sold around 120 sun / day. Now its being sold for 60 sun / day or even less.
> 
> Now concerning this proposal #560 , is Justlend planning to create a Bandwith Rental platform in the future?

Well, the recent low-value transactions should not be blamed on the rising energy price last year. This time, the main reason is large amount of accounts sending transactions using free bandwidth. Still I think reducing the daily free bandwidth is good way, others should remain unchanged.

**Comment by Dendorion** [comment]
As we warned during the Dynamic energy proposal this would be the result as most scammers switched to sending fake USDT transactions with TRC10 tokens. 

My suggestion would be drop all the free bandwidth make it 0 , as this proposal will result in them opening up more adresses to achieve the same result. And we will come back to a new proposal in a few months . Then after we can open the next proposal to increase energy fees again as they will circle back most likely. 

It's good people care so much in taking them out without thinking even the slightest on actuall chain users and dapps.

**Comment by simbadMarino** [comment]
You will never get rid of spam messages, even if we reduce the free bandwith to 0, scammers will rent bandwidth and get along with it. In my opinion it is better to work on UI developments on wallets to avoid users to be scammed and the network to be overloaded with obvious spam.

Leave the network as it is and focus on filters. I've been saying this. Do it like email clients! We receive tons of trash emails everyday but our clients do a pretty good job sending them to the junk bin, if we avoid normal users to see this kind of spam sooner or later the spamming addresses will stop doing this and network will be fine.

Please stop crushing the beauty of TRON which is the low cost and speed... :/

**Comment by simbadMarino** [comment]
@halibobo1205 , are we really struggling to process the spam transactions anyway? I mean from the network performance itself, is this really an issue?? I'm asking because I havent seen any network issue at all to justify this move? As I previously said, You will never get rid of spam attacks or low value transactions. If you are focusing on this you are losing your time. Spammers will keep spamming and will find  a way, ALWAYS.

**Comment by txoh1603** [comment]
@simbadMarino It is right that you can never get rid of scams, they will find a way to reach you. However, their way of utilizing daily free bandwidth to save their cost of sending scams can be handled in some way. That is why this TIP stands. 

Many of the wallets have already screened these low-value transactions or spamming transactions, which are really protective, but the scams are still able to address users other than wallets obviously. I agree with the author that this time some measures like this topic should be taken to target the resources that scams rely on, rather than target the channels that display.

**Comment by Dendorion** [comment]
This will result in the scammers creating more wallets to perform the same action . Set the free bandwidth to 0 so we do not have to revisit with another proposal in a few months . Then they can circle back to other tactics and the next proposal can be the increase of the energy price again. 

This is exactly what some of us predicted with the implementation of the Dynamic energy model . They will always find new ways , do not over extend without taking in account the effects this will have on Dapps and daily chain users . Of which we have seen both a drastic decrease already.

**Comment by txoh1603** [comment]
@Dendorion Thanks for bringing up the dynamic energy model, which has significantly cracked down scams with USDT and other TRC-20 tokens. Even though bandwidth takes a small part of transaction costs, it is not wise to just set the daily free bandwidth to 0. I think most of the users would like to keep this perk, making the network to be more friendly to newcomers. The TRON Network Parameters are instituted to be configurable, not just a simple on/off switch, for good purposes. Decent community governance is seasonable and subtle and should be done according to the circumstances in time.

How many Dapps and chain users seriously rely on the daily free to participate activities on chain? The free bandwidth should only be helpful in a timely fashion for a serious user. Modifying this has a tiny impact on normal users and Dapps.

**Comment by tronpulse** [comment]
> 

We don't see any difference to have free bandwidth for 1 tx or pay 0.27 TRX for new users. What is important for users is to keep the transactions fees low.

**Comment by Dendorion** [comment]
@txoh1603 Correct it has cracked down the scams significantly and made the scammers move to fake USDT token creations on trc10 using the free bandwidth model for them to create even cheaper transactions . Not setting it to 0 would result in them creating more wallets to have the same result . It is and will be an endless battle where scammers will adapt to the changes made on the network . And we will have to revisit a new proposal in a few months. They will not care to spend for instance $10k if the gain can be significantly higher.

Probably not many users still rely on this system you are correct as most low value users have already moved forward towards cheaper chains since all the drastic changes already implemented. But it is important to keep the costs low if we want to attract new users to the chain . 

Education of users is the key here not making the chain even more expensive to use.

**Comment by txoh1603** [comment]
@Dendorion That is exactly why this topic is dedicated to the free daily bandwidth, not the unit price yet. First, modify the free bandwidth and see how the stats change, and then other measures could be figured out accordingly. As you said, it is an endless battle, then please never count on some rules or principles that be set once and for all, or do not take any move at all. 

An eco-system grows based on proposal after proposal, that is the way how it survives. If they will create more wallets to have the same result, taking advantage of free bandwidth, they will care to spend $10k believe me.

**Comment by Dendorion** [comment]
@txoh1603 Seeing my prediction on previous proposals also came true i am not that easily convinced in believing . We have tracked down multiple of these scam wallets and found out several have earned upwards of $1 million or more by their actions . In case their new heist through trc10 becomes to expensive to execute they will simply switch back to their previous USDT methods. I guess in terms it will be good for the energy markets again as they will be getting much needed traffic but i am sure we are then looking again at a proposal to increase those costs. We clearly have a different view on things , for me the only real solution is education and awareness towards users.

**Comment by Jackmrshen** [comment]
> I made a comparison showing the relationship between free bandwidth and free transactions, please check it in the table below,
> 
> Daily free bandwidth	Daily number of free TRX transfers	Daily number of free TRC10 transfers	Daily number of free TRC20 transfers	Daily number of free TRC20 transferfroms
> 1500	5.60	5.26	3.97	4.35
> 700	2.61	2.46	1.85	2.03
> 600	2.24	2.11	1.59	1.74
> 500	1.87	1.75	1.32	1.45
> I think it would be enough for users including newbies to create 1 free transaction, which means cut the free bandwidth to 500 daily. This will decrease the low-value transactions to the most, while keep it possible for new comers to test a transaction on TRON.

Reducing or cancel the daily free bandwidth has the minimum impact on TRON normal users. I agree to start from here, and will give feedback about the consequence in time. 

@brooklynn1212 I noticed this table, as I checked previously the average daily transaction made by active account is 2, better set the value to still allow users make 2 trx or trc10 transactions. I prefer set the value daily free bandwidth value to 600.

**Comment by halibobo1205** [comment]
Thank you everyone for the comments on this proposal, and based on the discussions, I have updated the draft. The `totalnetlimit` and `transactionfee` will remain unchanged, only the `freenetlimit` will be adjusted from 1500 to 600. Thus, it is expected to decrease the large amount of low-value transactions, and meanwhile still support a user sending 2 free TRX/TRC10 transactions, or sending 1 free TRC20 transaction.

**Comment by hussaini2760** [comment]
Good step

**Comment by halibobo1205** [comment]
At present, since the proposal has been taken effect for a week, the data on-chain has achieved recovery to some degree:
1. The volume of transfer transactions with TRX < 0.1 has greatly decreased, the detailed comparison is shown below:

|| Daily last week after enabled | Daily last week before enabled | Daily last month before enabled | Daily last 3 months before enabled |
| -------- | -------- | -------- |--|--|
| Total TRX transfer transactions    | 1,996,165 | 5,054,115    | 4,586,870     | 4,409,105 |
| Transfer transactions with TRX < 0.1     | 1,075,224 | 4,006,043     | 3,559,799     | 3,321,912 |
2. There is also a significant decline in the volume of low-value smart contract triggers, as shown below:

|| Daily last weed after enabled | Daily last week before enabled | Daily last month before enabled | Daily last 3 months before enabled |
| -------- | -------- | -------- |--|--|
| Volume of smart contract triggers | 2,689,219 | 5,244,816    | 3,478,842    | 2,618,377 |
| Volume of low-value transactions     | 1,280,184 | 3,661,780 | 2,026,656 | 1,174,615 |

To sum up, this proposal has met its expectation, I will continue to follow up the data on-chain and bring you more effects of this proposal.

**Comment by halibobo1205** [comment]
Thanks to everyone’s contribution to this proposal.
This issue will be closed as it is already going to effect, check detail at: https://tronscan.org/#/proposal/87

**Comment by Blonde488** [comment]
hope you like this!!

**Comment by Natchaphonwongplianphan** [comment]
🇹🇭🙏🇹🇭

รับ Outlook สำหรับ Android<https://aka.ms/AAb9ysg>
________________________________
